src/emulator/functionsEmulator.ts

```diff
@@ -435,8 +441,10 @@ You can probably fix this by running "npm install ${
     // TODO: This call requires authentication, which we should remove eventually
     this.firebaseConfig = await functionsConfig.getFirebaseConfig(this.options);
 
+    const host = this.getInfo().host;
     this.server = FunctionsEmulator.createHubServer(this.bundleTemplate, this.nodeBinary).listen(
-      this.port
+      this.port,
```

Comment:
Should this now be `getInfo().port`?

Also maybe just do `const {host, port} = this.getInfo();`?